packages/atomic/src/components/facets/atomic-segmented-facet/atomic-segmented-facet.tsx

```diff
@@ -146,9 +150,19 @@ export class AtomicSegmentedFacet
   }
 
   public render() {
+    if (
+      this.searchStatus.state.hasError ||
+      !this.facetState.values.length ||
+      !this.facet.state.enabled
+    ) {
+      return <Hidden></Hidden>;
+    }
+
     return (
       <div class="flex">
-        <b class="inline-block my-3 mr-2">{this.label}</b>
+        <b class="inline-block my-3 mr-2">
+          {this.label !== undefined ? this.label.concat(':') : ''}
+        </b>
```

Comment:
If there's no label, I would remove the `b` element entirely, especially since it has a margin.

```suggestion
        {this.renderLabel()}
```
...
```ts
public renderLabel() {
  if (!this.label) {
    return;
  }
  return (
    <b class="inline-block my-3 mr-2">
      {this.label.concat(':')}
    </b>
  );
}
```